socs/agent/smurf_recorder.py

```diff
@@ -162,13 +177,16 @@ def __init__(self, file_duration, tcp_addr, data_dir, stream_id,
         # Parameters
         self.time_per_file = file_duration
         self.address = tcp_addr
+        self.host, self.port = self.address[6:].split(':')
```

Comment:
Either add these to the Attributes list in the docstring, or better yet make `check_port()` take the format stored in `self.address`.